frontend/src/v4/routes/viewerGui/components/previewDetails/previewDetails.component.tsx

```diff
@@ -255,22 +255,22 @@ export class PreviewDetails extends PureComponent<IProps, any> {
 				>
 					<RoleIndicator color={roleColor} ref={this.headerRef} />
 					<MainInfoContainer>
-						<Grid container alignItems="center" >
+						<Grid container alignItems="center">
 							{this.renderTitleNumber(number)}
 							{this.renderName(!editable)}
 							{this.renderNameField(editable)}
 						</Grid>
-							<PreviewItemInfo
-								author={owner}
-								createdAt={created}
-								StatusIconComponent={StatusIconComponent}
-								statusColor={statusColor}
-								actionButton={actionButton}
-								showModelButton={showModelButton}
-								type={type}
-								id={id}
-								urlParams={urlParams}
-							/>
+						<PreviewItemInfo
+							author={owner}
```

Comment:
The author doesn't appear when you are creating an issue

Reply:
fixed

Reply:
fixed